plugins/gitignore/gitignore.plugin.zsh

```diff
@@ -1,4 +1,4 @@
-function gi() { curl -sL https://www.gitignore.io/api/$@ ;}
+function gi() { curl -sL https://www.gitignore.io/api/`echo $@ | sed s/" "/,/g` ;}
```

Comment:
Whoa killer stuff.Not too familiar with zsh specific syntax and wouldn't have even thought of something that awesome